### ISSUE TITLE
Questions about training epochs and evaluation metrics on ScanNet

### ISSUE THREAD

**Issue by jinwensheng18** [vLAR-group/GrowSP#32]
Hi, thank you very much for open-sourcing this excellent project. I really appreciate your work on GrowSP.

I have two questions regarding the ScanNet training and evaluation details, and I would be very grateful if you could help clarify them.

1. Training epochs discrepancy
In the paper, it is mentioned that:  “We train a total of 800 epochs. Since ScanNet has an undefined category which does not have consistent geometry, we exclude (mask) these points during training.” 
However, in the provided training script train_ScanNet.py, the default configuration is:
`parser.add_argument('--max_epoch', type=list, default=[350, 600], help='max epoch')
`
This seems to correspond to a total of 950 epochs (350 epochs before superpoint growing and 600 epochs after), which does not exactly match the 800 epochs described in the paper.

Could you please clarify:

- Whether the reported 800 epochs refers to an effective number of epochs (e.g., excluding some warm-up or overlapping stages)?
- Or whether the training schedule in the released code differs slightly from the one used in the paper?

2.  Evaluation protocol and reported metric ranges on ScanNet 
In the paper, the ScanNet evaluation results are reported as ranges, for example:
OA (%): 57.3 ± 2.3
 mAcc (%): 44.2 ± 3.1
 mIoU (%): 25.4 ± 2.3
I would like to better understand how these metrics are computed in practice:

- Are these ranges obtained by running multiple training trials with different random seeds?

- Are the metrics computed over all valid classes excluding the undefined category?

- Is the evaluation performed at the point level or superpoint level?

- Which evaluation script or protocol should be used to reproduce these numbers?

Any clarification on the exact evaluation pipeline would be very helpful for reproducing the reported results.
Thank you very much for your time and for making this work publicly available!
Best regards

**Comment by zihui0930** [comment]
Hi, 

800 epochs are enough for ScanNet training, we often train some more epochs, as written in the paper, but 800 is enough for convergence. The metric range is computed by the last 50 epochs, which has been written in Table 1 in our paper. All metrics are computed on the point level, we report the per-category results in the supplementary.
